.github/actions/get-changed-gha-items/Get-GhaItemsFromFileList.ps1

```diff
@@ -4,18 +4,38 @@ param(
 
 # Filter actions based on files in action directory.
 [array]$actionFiles = $FileIncludeList | Where-Object -FilterScript {
-    (Get-Item $PSitem).Directory.GetFiles('action.yml').Count -gt 0 -or
-    (Get-Item $PSitem).Directory.GetFiles('action.yaml').Count -gt 0
+    try
+    {
+        $itemDirectory = (Get-Item $PSItem).Directory.FullName
+        $isInGitHubDir = $itemDirectory -like '*/.github/*' -or $itemDirectory -eq '*/.github'
+        if (-not $isInGitHubDir) {
+            return $false
```

Comment:
This will not cause the action thus the workflow to fail, right?

Reply:
Looks like it run ok [here](https://github.com/Lombiq/GitHub-Actions/actions/runs/7646906552/job/20836729932?pr=302), correct?